zerver/views/typing.py

```diff
@@ -0,0 +1,20 @@
+from django.http import HttpRequest, HttpResponse
+from six import text_type
+
+from zerver.decorator import authenticated_json_post_view,\
+    has_request_variables, REQ, JsonableError
+from zerver.lib.actions import check_send_typing_notification, \
+    extract_recipients
+from zerver.lib.response import json_success
+from zerver.models import UserProfile
+
+@has_request_variables
+def send_typing_notification_backend(request, user_profile,
+                                     operator = REQ('op'),
+                                     notification_to = REQ('to', \
+                                            converter=extract_recipients, \
+                                            default=[])):
```

Comment:
This line-wrapping feels awkward.  A few possible solutions:
* Remove "typing_" from the function name, since it's clear from the file?
* we usually allow lines up to ~100 characters when it makes the code more readable, so just wrap it less aggressively without the `\`s.